addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -81,7 +90,8 @@ odoo.define('website.tour.form_editor', function (require) {
             });
         }
         if (label) {
-            testText += `:has(label:contains("${label}"))`;
+            testText += label.includes('"') ? `:has(label:contains('${label}'))`
```

Comment:
Can't we just remove the `"` usage and so not need this diff?
Is there any difference between `:contains('ok')` and `:contains(ok)`

Reply:
In fact, I had to add this diff because I wanted to test all the cases with special characters (`"`, `'` and ``` ` ```). So in the case of the `"` character, it has to be `:has(label:contains('${label}'))` (as `:has(label:contains("""))` would throw an error) and for the `'` character, it has to be `:has(label:contains("${label}"))` (as `:has(label:contains('''))` would throw an error). But sure, if I remove the test of the `"` character as a label then I can remove this this :+1:

Reply:
Yes please, it not relevant in the test anyway, it could have been written like that from start.
I don't even think using `'` or `"` inside `contains()` in jquery is the recommended way (probably for this reason).